src/renderers/render.js

```diff
@@ -1,24 +1,36 @@
 import { buildComponentFromVNode } from '../isomorphic/createComponent';
-import RenderUtils from '../utils/render-utils';
 
-let FiberComponent;
+let component;
+let initialVnode;
```

Comment:
Not sure I got this right, but are initialVnode and component are shared across calls to render() ?

Reply:
Hey @brunobasto, The names are not clear at all 😅.

The `component` is used to store the instance of the component created in `buildComponentFromVNode`, perhaps it will be clearer in #44.

Reply:
In the case of `component` it is more to have an instance of the component to be used at the time of creating events when passing an `onClick` for example.